statsmodels/tsa/adfvalues.py

```diff
@@ -223,127 +273,138 @@ def mackinnonp(teststat, regression="c", N=1, lags=None):
-                  [-4.96397,-19.6064,-18.858,174.919]],
-                [ [-6.00609,-38.7383,-108.605,365.208], # N = 7
-                  [-5.47664,-27.3005,-39.498,246.918],
-                  [-5.19921,-22.2617,-17.910,208.494]],
-                [ [-6.22758,-42.7154,-119.622,421.395], # N = 8
-                  [-5.69983,-30.4365,-44.300,345.48],
-                  [-5.42320,-24.9686,-19.688,274.462]],
-                [ [-6.43933,-46.7581,-136.691,651.38],  # N = 9
-                  [-5.91298,-33.7584,-42.686,346.629],
-                  [-5.63704,-27.8965,-13.880,236.975]],
-                [ [-6.64235,-50.9783,-145.462,752.228], # N = 10
-                  [-6.11753,-37.056,-48.719,473.905],
-                  [-5.84215,-30.8119,-14.938,316.006]],
-                [ [-6.83743,-55.2861,-152.651,792.577], # N = 11
-                  [-6.31396,-40.5507,-46.771,487.185],
-                  [-6.03921,-33.8950,-9.122,285.164]],
-                [ [-7.02582,-59.6037,-166.368,989.879], # N = 12
-                  [-6.50353,-44.0797,-47.242,543.889],
-                  [-6.22941,-36.9673,-10.868,418.414]]]
+tau_ctt_2010 = [
+    [[-4.37113, -11.5882, -35.819, -334.047],  # N = 1
+     [-3.83239, -5.9057, -12.490, -118.284],
+     [-3.55326, -3.6596, -5.293, -63.559]],
+    [[-4.69276, -20.2284, -64.919, 88.884],    # N =2
+     [-4.15387, -13.3114, -28.402, 72.741],
+     [-3.87346, -10.4637, -17.408, 66.313]],
+    [[-4.99071, -23.5873, -76.924, 184.782],   # N = 3
+     [-4.45311, -15.7732, -32.316, 122.705],
+     [-4.17280, -12.4909, -17.912, 83.285]],
+    [[-5.26780, -27.2836, -78.971, 137.871],   # N = 4
+     [-4.73244, -18.4833, -31.875, 111.817],
+     [-4.45268, -14.7199, -17.969, 101.92]],
+    [[-5.52826, -30.9051, -92.490, 248.096],   # N = 5
+     [-4.99491, -21.2360, -37.685, 194.208],
+     [-4.71587, -17.0820, -18.631, 136.672]],
+    [[-5.77379, -34.7010, -105.937, 393.991],  # N = 6
+     [-5.24217, -24.2177, -39.153, 232.528],
+     [-4.96397, -19.6064, -18.858, 174.919]],
+    [[-6.00609, -38.7383, -108.605, 365.208],  # N = 7
+     [-5.47664, -27.3005, -39.498, 246.918],
+     [-5.19921, -22.2617, -17.910, 208.494]],
+    [[-6.22758, -42.7154, -119.622, 421.395],  # N = 8
+     [-5.69983, -30.4365, -44.300, 345.48],
+     [-5.42320, -24.9686, -19.688, 274.462]],
+    [[-6.43933, -46.7581, -136.691, 651.38],   # N = 9
+     [-5.91298, -33.7584, -42.686, 346.629],
+     [-5.63704, -27.8965, -13.880, 236.975]],
+    [[-6.64235, -50.9783, -145.462, 752.228],  # N = 10
+     [-6.11753, -37.056, -48.719, 473.905],
+     [-5.84215, -30.8119, -14.938, 316.006]],
+    [[-6.83743, -55.2861, -152.651, 792.577],  # N = 11
+     [-6.31396, -40.5507, -46.771, 487.185],
+     [-6.03921, -33.8950, -9.122, 285.164]],
+    [[-7.02582, -59.6037, -166.368, 989.879],  # N = 12
+     [-6.50353, -44.0797, -47.242, 543.889],
+     [-6.22941, -36.9673, -10.868, 418.414]]]
 tau_ctt_2010 = asarray(tau_ctt_2010)
 
+_tau_2010s = {
```

Comment:
Why reason why this is _? The rest of the tau variable's aren't, and there is an __all__ to help.

Reply:
Accident.  I lean towards privatizing the other, am OK with making this one public.  Any preference?